src/components/QuranReader/TafsirView/TafsirBody.tsx

```diff
@@ -222,6 +223,35 @@ const TafsirBody = ({
           render={renderTafsir}
         />
       )}
+
+      <TafsirEndOfScrollingActions
+        currentVerseNumber={selectedVerseNumber}
+        currentChapterId={selectedChapterId}
+        onNextButtonClicked={() => {
```

Comment:
will do a follow up to scroll the page to the top when user click 'next' or 'prev'

Reply:
Add logging